pkg/kapp/config/default.go

```diff
@@ -386,10 +386,18 @@ changeGroupBindings:
   resourceMatchers: &crdMatchers
   - apiGroupKindMatcher: {kind: CustomResourceDefinition, apiGroup: apiextensions.k8s.io}
 
+- name: change-groups.kapp.k14s.io/crds-{crd-group}-{name}
+  resourceMatchers: &crdMatchers
+  - apiGroupKindMatcher: {kind: CustomResourceDefinition, apiGroup: apiextensions.k8s.io}
+
 - name: change-groups.kapp.k14s.io/namespaces
   resourceMatchers: &namespaceMatchers
   - apiGroupKindMatcher: {kind: Namespace, apiGroup: ""}
 
+- name: change-groups.kapp.k14s.io/namespaces-{name}
+  resourceMatchers: &namespaceMatchers
```

Comment:
We can reuse namespaceMatcher from above instead of redeclaring again.

Reply:
Makes sense, will make the change.

Reply:
Resolved in amended commit